src/main/kotlin/racingcar/domain/model/CarRacer.kt

```diff
@@ -0,0 +1,28 @@
+package racingcar.domain.model
+
+/**
+ * 자동차 경주자
+ */
+class CarRacer(val name: String, position: Int = 0) {
+
+    var position: Int = position
+        private set
+
+    init {
+        require(name.length <= 5) { "자동차 이름은 5자를 초과할 수 없습니다." }
+    }
+
+    fun moveTo(count: Int) {
+        if (count < 4) return
+        position++
+    }
```

Comment:
부등호의 위치를 바꿔주면 return 이 제거되겠네요 😄 
```suggestion
    fun moveTo(count: Int) {
        if (count >= 4) position++
    }
```

Reply:
리턴을 제거합니다!